packages/fxa-payments-server/src/routes/Product/SubscriptionCreate/index.test.tsx

```diff
@@ -164,9 +162,20 @@ describe('routes/Product/SubscriptionCreate', () => {
         />
       );
     });
-    waitForExpect(() =>
-      expect(queryByTestId('paypal-button')).toBeInTheDocument()
+    await waitForExpect(() => {
+      expect(queryByTestId('paypal-button')).toBeInTheDocument();
+    });
```

Comment:
hmm `act` _should_ wait for all rendering to be done so the additional waiting doesn't seem necessary here.  (But I didn't actually debug here so maybe not?)

Reply:
Strangely in this case it was necessary. Not sure why, but when I removed it, the test started failing.